atd-vze/src/views/VZDashboard/VZDashboard.js

```diff
@@ -140,6 +140,14 @@ function VZDashboard() {
             link="https://visionzero.austin.gov/viewer/"
             target="_vzv"
           />
+          <VZLinksWidget
+            header={`Access Management Crashes`}
+            mainText={`Summary of crashes for individual locations`}
+            icon="fa fa-map"
+            color="primary"
+            link="https://austin.maps.arcgis.com/apps/instant/interactivelegend/index.html?appid=ea84226e6e5a4ecf998082b73b8c6cca"
+            target="_arcgis"
```

Comment:
can anyone explain what `target="_arcgis"` does? this is just out of my own curiosity lol. i tried looking up the target attribute for the `a` tag and read about the diff values [here](https://www.w3schools.com/tags/att_a_target.asp). you can specify your own value like we do here to open the link in a defined iframe but we dont use any iframes afaik

Reply:
Rose, tbh I don't know what `target="_arcgis"` is. I was following the nomenclature of the other targets listed. ex: `_word`. 

I'm not sure if we have established targets or if there's one that describes the widget more clearly. If we do, I'm happy to change `_arcgis` to what it's supposed to be.

---

<img width="460" alt="Screen Shot 2023-03-28 at 5 14 16 PM" src="https://user-images.githubusercontent.com/5566727/228379100-e6b8e180-4b06-4fd2-815e-1496c0ba7ebd.png">